.gitignore

```diff
@@ -80,3 +80,35 @@ ircmaxell/random-lib/composer.lock
 ircmaxell/random-lib/phpunit.xml.dist
 
 icewind/streams/tests
+
+guzzlehttp/guzzle/build/
+guzzlehttp/guzzle/docs/
+guzzlehttp/guzzle/tests/
+guzzlehttp/guzzle/.gitignore
+guzzlehttp/guzzle/.travis.yml
+guzzlehttp/guzzle/CHANGELOG.md
+guzzlehttp/guzzle/composer.json
+guzzlehttp/guzzle/LICENSE
```

Comment:
I have no idea, but that is just a feeling (because the files itself doesn't contain a license header)